{
	"schema_version": "1.6.0",
	"id": "PUB-A-341120728",
	"modified": "2024-10-10T15:12:10.966571Z",
	"published": "2024-10-01T00:00:00Z",
	"aliases": [
		"A-341120728",
		"CVE-2024-47035"
	],
	"details": "In vring_init of external/headers/include/virtio/virtio_ring.h, there is a possible out of bounds write due to a logic error in the code. This could lead to local escalation of privilege with no additional execution privileges needed. User interaction is not needed for exploitation.",
	"affected": [
		{
			"package": {
				"ecosystem": "Android",
				"name": ":unknown:"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "Pixel-family specific:0"
						},
						{
							"fixed": "Pixel-family specific:2024-10-05"
						}
					]
				}
			],
			"versions": [
				"Pixel-family specific"
			],
			"ecosystem_specific": {
				"severity": "Critical",
				"spl": "2024-10-05",
				"types": [
					"EoP"
				]
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/android-osv/PUB-A-341120728.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://source.android.com/security/bulletin/2024-10-01"
		}
	]
}
